### DISCUSSION TITLE
Custom domain made page inaccessible

### DISCUSSION THREAD

**Discussion by RadriaansNES** [community/community#64977] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi guys, I've been working on a website pretty religiously and I just tried to set a custom domain name to the page instead of the usual github.io etc one. 

I switched it, then when I clicked on the link, it said something about a bad domain name. I figured I wouldn't sweat it as it's just being developed now, and as such clicked remove custom domain name. 

Now, when I try and access the page from the github page section where the link is shown or from just typing it in, it re-directs to the custom domain I tried using earlier and blocks me from accessing the page. 

Here is the link to the repository: https://github.com/RadriaansNES/ECom
Here is the link to the page: https://radriaansnes.github.io/ECom/

Please help as this really sucks being unable to access it now.

**Comment by queenofcorgis** [comment, marked as answer]
@RadriaansNES hey there 👋🏼 ! 

I know this may sound simple, but have you tried clearing your cache? When you change or remove your custom domain and can't access the new URL in your browser, you may need to clear your browser's cache to reach the new URL

**Reply by RadriaansNES** [reply]
Hi queen, that's exactly what it ended up being. Thanks!!

**Reply by queenofcorgis** [reply]
@RadriaansNES you're welcome, I'm so glad it was an easy fix 💫  Thanks for letting me know it worked and marking the answer ✅
